css/style.css

```diff
@@ -0,0 +1,532 @@
+  border: 4px solid yellow;
+}
+
+.header-logo {
+  height: 40px;
+}
+
+ul,
+ul li {
+  list-style-type: none;
+}
+
+.header-container {
+  max-height: 90px;
+  background-color: #ff0500;
+  height: 100px;
+}
+
+.page-header {
+  z-index: 2;
+  left: 33%;
+}
+
+/* This is for the mobile navbar */
+.mobile-navbar {
+  background: #f72210;
+  height: 50px;
+}
+
+.navbar-brand {
+  font-size: 2rem;
+  font-weight: 700;
+}
+
+.caption {
+  text-align: center;
+  position: relative;
+  top: -35px;
+  left: 0;
+  background-color: #fff;
+  width: 80px;
+  height: 35px;
+  opacity: 0.9;
+}
+
+.caption a,
+.caption1 a,
+.caption-culture a {
+  color: red;
+  align-self: center;
+  font-weight: 800;
+  font-size: 1em;
+}
+
+.story-grid a {
+  color: #939393;
+  font-weight: bold;
+}
+
+nav > div.w-30 {
+  height: inherit;
+  width: 30%;
+}
+
+.block-title > a {
+  line-height: 2.25rem;
+}
+
+nav > div.w-30 > a {
+  font-size: 1.5rem;
+}
+
+nav > div.w-70 {
+  height: inherit;
+  width: 70%;
+}
+
+.form-inline .form-control {
+  width: 64px;
+}
+
+.email-subscription-top {
+  display: none;
+}
+
+.email-subscription-secret {
+  width: 70%;
+  height: auto;
+  background-color: #f4f5f6;
+}
+
+/* Main Section Starts here */
+
+main {
+  margin-top: 50px;
+  height: auto;
+  width: 100%;
+}
+
+article.subscribe-art {
+  height: 850px;
+}
+
+.main-backg {
+  background: url("https://d.newsweek.com/en/full/1610540/chad-wolf.jpg?w=591&h=364&f=c8f9c43163fe20e3117d80cd9b3d980a");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 243px;
+}
+
+div.mag-thumb.current,
+#newsletter-img-top {
+  z-index: 6;
+  top: -210px;
+  left: 90px;
+  position: relative;
+}
+
+div.sub-left {
+  height: 280px;
+}
+
+div.last-section {
+  height: 800px;
+  margin: 0;
+}
+
+div.fn.image {
+  height: 280px;
+}
+
+div.f12.info {
+  position: absolute;
+  z-index: 7;
+  top: 650px;
+}
+
+article.newsletter {
+  position: relative;
+  z-index: 7;
+  top: 20px;
+}
+
+article {
+  height: auto;
+}
+
+h3 {
+  font-size: 24px;
+  line-height: 24px;
+}
+
+h5.mini-header {
+  font-size: 16px;
+  line-height: 16px;
+  font-weight: bold;
+}
+
+.image-thumbnail {
+  width: 63px;
+}
+
+#versus-logo {
+  width: 30px;
+}
+
+.email-subscription,
+.email-subscription-top {
+  background-color: #f4f5f6;
+}
+
+#versus-container::before {
+  content: "";
+  display: block;
+  background-color: #d9d9d9;
+  height: 1px;
+  position: absolute;
+  left: 0;
+  right: 0;
+}
+
+#versus-container::after {
+  content: "";
+  display: block;
+  background-color: #d9d9d9;
+  height: 1px;
+  position: absolute;
+  left: 0;
+  right: 0;
+}
+
+.horizontal-line {
+  z-index: 1;
+  top: 17px;
+  height: 3px;
+}
+
+.svg-image {
+  z-index: 50;
+  top: 0;
+}
+
+.section-logo {
+  width: 29px;
+}
+
+#newsletter-img-top {
+  width: 177px;
+}
+
+#footer-logo {
+  width: 251px;
+}
+
+.fa {
+  color: #656665;
+}
+
+#nl-img-cont {
+  height: 180px;
+  padding-left: 10%;
+}
```

Comment:
- [ ] A padding left of 25% and a padding top, would be enough for centering this image. Please add it
![image](https://user-images.githubusercontent.com/53256139/89106461-abba9e00-d3ef-11ea-89bd-ca97447fab1b.png)

Reply:
Issue fixed